src/helpers/utils.tsx

```diff
@@ -983,3 +983,28 @@ export const creatSettingsPayloads = (result: PapaResult): SettingConfiguration[
   }
   return payloads;
 };
+
+/** Function to download data to a file
+ * @param {string} data - data to be written to file
+ * @param {string} filename - name of the file to be saved
+ * @param {string} type - MIME type for the file
+ */
+export const downloadFile = (data: string, filename: string, type: string) => {
```

Comment:
@moshthepitt The type passed here determines the file type. In this case I pass text/csv which downloads a csv file

Reply:
@gmwasi thanks for clarifying.  What I mean is that I also would like the file to have a .csv extension e.g. instead of `Jurisdiction Metadata` let us have something like `Jurisdiction Metadata.csv`

Reply:
Please create an issue to fix this for Linux.  I am consistently getting a file named `Jurisdiction Metadata` instead of `Jurisdiction Metadata.csv`